pkg/event/router.go

```diff
@@ -0,0 +1,11 @@
+package event
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func Routes(r *gin.Engine, authenticator gin.HandlerFunc, handler Handler) {
+	queryStringAuthenticationRouter := r.Group("")
```

Comment:
This is a bit of spillover from other places where we call it tokenAuthenticatedRouter. The type depends on the authenticator we're passing to the Routes function main. However this doesn't matter much here since we're only dealing with one type of authentication and one route. So I guess we could just name the variable route which I've done for now. Unless you have any objections, please mark this conversation as resolved.